test/e2e/operator/main_test.go

```diff
@@ -24,11 +24,10 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
+	ginkgotypes "github.com/onsi/ginkgo/v2/types"
```

Comment:
can be dot import or gives a conflict ?
or maybe even nameless `"github.com/onsi/ginkgo/v2/types"`

Reply:
We cannot import types with dot
```
test/e2e/handler/main_test.go:28:2: `Report` redeclared in this block (typecheck)
	. "github.com/onsi/ginkgo/v2/types"
```